### ISSUE TITLE
Typos; Editorial tweaks

### ISSUE THREAD

**Issue by richb-hanover** [eitsupi/querying-with-prql#1]
This looks to be a terrific integration between PRQL and DuckDB. I noticed some bits of fluff...

* The heading "Quering with PRQL" needs a "y" in "Querying"
* The intro paragraph is a little tentative. To make a stronger introduction, I would reword it like this (skip the "Introduction" paragraph):

> Motivation
> 
> I wanted to compare PRQL and the latest DuckDB SQL (and other query libraries) to clarify the missing features of PRQL and highlight the latest features of DuckDB.
>
> * [PRQL](https://prql-lang.org/) is a modern language for data manipulation that compiles to SQL. Although the PRQL language is still evolving, it is easy to create complex SQL queries. 
> * [DuckDB](https://duckdb.org/) is a modern in-memory SQL OLAP ([very fast](https://duckdblabs.github.io/db-benchmark/)) database management system. DuckDB is also evolving, with new features and improving syntax.
> 
> This project uses CLI, JavaScript, Python, R, etc. to combine the PRQL compiler and DuckDB database. This gives a powerful database that can be queried with a simple syntax.
> 
> Content  & Credit
> 
> ... the rest is OK...

(I hope I got your meaning correct...) Thanks.

**Comment by eitsupi** [comment]
Thanks for opening this!
If you don't mind, could you send a PR?

You don't need Quarto or Docusaurus to update the text only (you don't need to check if the website build is successful)

**Comment by eitsupi** [comment]
I think this is fixed by d2da2536a0d635cb810287aefcc9b9a8e17d3607
